Makefile

```diff
@@ -112,8 +112,8 @@ deploy-full-local-setup: ## Deploy full local multicluster setup (k3d >= 4.2.0)
 .PHONY: deploy-stable
 deploy-stable:
 	@echo "\n$(YELLOW) import $(CYAN)k8gb:$(VERSION) $(YELLOW)to $(CYAN)$(CLUSTER_GSLB1), $(CLUSTER_GSLB2) $(NC)"
-	$(call deploy-local-cluster,$(CLUSTER_GSLB1),$(CLUSTER_GSLB2),$(VERSION),,'k8gb/k8gb')
-	$(call deploy-local-cluster,$(CLUSTER_GSLB2),$(CLUSTER_GSLB1),$(VERSION),$(CLUSTER_GSLB2_HELM_ARGS),'k8gb/k8gb')
+	$(call deploy-local-cluster,$(CLUSTER_GSLB1),$(CLUSTER_GSLB2),,,'k8gb/k8gb')
+	$(call deploy-local-cluster,$(CLUSTER_GSLB2),$(CLUSTER_GSLB1),,$(CLUSTER_GSLB2_HELM_ARGS),'k8gb/k8gb')
```

Comment:
https://github.com/AbsaOSS/k8gb/blob/bf5bc287c207326cc70bd9ab164f3ed34176f8c8/Makefile#L353
will be confusingly empty
Consider changing
```
@echo "\n$(YELLOW)Deploy GSLB operator from $3 $(NC)"
```
to
```
@echo "\n$(YELLOW)Deploy GSLB operator from ${3:-stable chart} $(NC)"
```

Reply:
@ytsarev thanks for suggestion, amended

Reply:
https://github.com/AbsaOSS/k8gb/pull/471/checks?check_run_id=2534570627#step:5:17 unfortunately it takes empty string as a value

Reply:
@ytsarev which is fine, it takes "released" image then.

Reply:
@k0da i'm talking about empty value in the log statement

Reply:
@ytsarev is it confusing? I think making it conditional will complicates make file quite a bit. I don't see harm setting a value to be empty.

Reply:
@k0da 
![image](https://user-images.githubusercontent.com/518532/117638992-baef2900-b183-11eb-924f-1c61b1769f9d.png)
Yes, `Deploy GSLB operator from <nothing>` is a confusing incomplete statement.